src/js/lib/ics.js

```diff
@@ -58,6 +59,10 @@ var ics = function(uidDomain, prodId) {
         return false;
       }
 
+      // if status is null/undefined, default to "busy"
+      if (status == null)
+        status = CalendarEventStatus.BUSY;
+
```

Comment:
```suggestion
      // if status is null/undefined, default to "busy"
      status = status || CalendarEventStatus.BUSY;
```
totally optional suggestion I sometimes use for these defaults, it supposedly has another assign but that should not be an issue 🤷